README.md

```diff
@@ -1,8 +1,12 @@
+[![Build Status](https://travis-ci.org/jdesrosiers/todo-api.svg?branch=master)](https://travis-ci.org/jdesrosiers/todo-api)
+
 TODO API
 ========
 
 The TODO API was created to showcase a Hypermedia API.
 
+**Note**: This application is deployed to Heroku, at https://hypermedia-todo.herokuapp.com/
```

Comment:
I was hoping for a little more that this. Deployment deserves it's own section in the README. If I want to know how the app is deployed, this should answer all my questions. Mention that we are using a continuous deployment strategy and what that means. Mention that all changes are immediately and automatically deployed as soon as they are merged to master. Mention that deployment should never be done manually, only through the automated process.

Reply:
@jdesrosiers, changes have been made. Please let me know what other comments you have.

Reply:
@jdesrosiers, changes have been made. Please let me know what other comments you have.